lumeer-storage/lumeer-storage-mongodb/src/main/java/io/lumeer/storage/mongodb/util/MongoFilters.java

```diff
@@ -85,4 +86,14 @@ public static Bson suggestionsFilter(SuggestionQuery query) {
       );
    }
 
+   public static <T> void addPaginationToSuggestionQuery(FindIterable<T> findIterable, SuggestionQuery query) {
```

Comment:
I don't think this method belongs to this class. I would suggest moving it `MongoDao` abstract class for now. And we will see how many similar methods will get there and can then extract them to a new class.

Reply:
I thought you would not like it here 😄